### PR TITLE
[ANCHOR-704] Document user_action_required_by

### PR DESCRIPTION
**Pull request by JiahuiWho**
1. Clean up deprecated `amount_fee` and update platform transaction schema
2. Add user_action_required_by to related RPC methods
    NotifyInteractiveFlowCompletedRequest
    NotifyTransactionOnHoldRequest
    NotifyOffchainFundsPendingRequest 
    NotifyOffchainFundsAvailableRequest
    RequestCustomerInfoUpdateRequest
    RequestOffchainFundsRequest
    RequestOnchainFundsRequest
    RequestTrustRequest
4. Add use instruction in SEP-6 and SEP-24
    platforms/anchor-platform/admin-guide/sep6/configuration.mdx
    platforms/anchor-platform/admin-guide/sep24/configuration.mdx

### CODE REVIEW

**Comment by lijamie98** [comment]
There are 8 actions modified. But the PR has 148 files modified. Is that expected?

**Comment by philipliu** [comment]
I think `yarn api` regenerated the API docs for SDP and Horizon. Can we remove those from the PR to make it easier to review?

**Comment by JiahuiWho** [comment]
There will me more files than just 8 because cleaning up `amount_fee ` from all existing rpc methos

File: platforms/anchor-platform/admin-guide/sep24/configuration.mdx
Comment (philipliu) [inline platforms/anchor-platform/admin-guide/sep24/configuration.mdx:117]:
```suggestion

`SEP24_INITIAL_USER_DEADLINE_SECONDS` is the time in seconds a user has to act before the transaction automatically moves to the next status. It determines the `user_action_required_by` field, which indicates the deadline for statuses with automatic movement. Check [JSON-RPC Methods][json-rpc-methods] for usage examples.
```

File: platforms/anchor-platform/admin-guide/sep6/configuration.mdx
Comment (philipliu) [inline platforms/anchor-platform/admin-guide/sep6/configuration.mdx:167]: Should we be linking to the callback docs instead of `sep38`?

**Review by philipliu — APPROVED** [review]
LGTM. Let's get one more review though.

**Comment by Ifropc** [comment]
We should not merge this PR before https://github.com/stellar/stellar-docs/pull/872 
There's going to be a lot of merge conflicts as the result, and I think it's better to resolve conflicts here than there (it's already a huge change in Elliot's PR)

**Comment by JiahuiWho** [comment]
Closing this one to create a new PR to avoid mass merge

**Pull request closed without merging**
